packages/frontend/src/lib/upstream/Link.svelte

```diff
@@ -23,8 +24,7 @@ function click() {
   if (internalRef) {
     router.goto(internalRef);
   } else if (externalRef) {
-    // TODO: Does not work at the moment
```

Comment:
Can't access PD API from within the frontend, I think this is expected.

Reply:
Have added this as a new API command / fixed the "upstream" code. We had to add the API call for externalLink for it to work: https://github.com/containers/podman-desktop-extension-bootc/pull/241/files#diff-db53f7a2ddea88bd5039df3b77b7bf1141e88c3c05858eb035f5ce05a1c1e35bR31